frontend/apps/reader/modules/readermenu.lua

```diff
@@ -184,6 +186,20 @@ function ReaderMenu:setUpdateItemTable()
     self.tab_item_table = MenuSorter:mergeAndSort("reader", self.menu_items, order)
 end
 
+function ReaderMenu:exitOrRestart(callback)
+    self:onTapCloseMenu()
+    UIManager:scheduleIn(0.1, function()
```

Comment:
This is a copy-paste from original logic, I was also curious about the scheduleIn(0.1, ...).
If you also prefer to use nextTick, I am happy to do so.